ingest/Ingest.py

```diff
@@ -0,0 +1,102 @@
+__author__ = 'Hao'
+
+import multiprocessing
+from SPARQLWrapper import SPARQLWrapper, JSON
+import itertools
+import json
+import requests
+
+class Ingest:
+    """say something"""
```

Comment:
This is a class docstring.  The value isn't helpful, but the string immediately following a class or function defintions is a python docstring.

http://epydoc.sourceforge.net/docstrings.html